This fixes

/home/janneke/vc/gub-samco/target/linux-arm-vfp/gubfiles/build/busybox-1.2.2.1/include/bbconfigopts.h:13 hmm, unterminated

--- ./scripts/config/mkconfigs~	2006-07-01 00:42:01.000000000 +0200
+++ ./scripts/config/mkconfigs	2007-06-01 10:19:46.000000000 +0200
@@ -46,6 +46,6 @@
  */"
 
 echo "static const char * const bbconfig_config ="
-echo "`sed 's/\"/\\\\\"/g' $config | grep "^#\? \?CONFIG_" | awk '{print "\\"" $0 "\\\\n\\"";}'`"
+echo "`sed 's/\"/\\\\\"/g' $config | grep "^#\? \?CONFIG_" | awk '{print "\\"" $0 "\\"";}'`"
 echo ";"
 echo "#endif /* _BBCONFIGOPTS_H */"
